### PR TITLE
Add documentation about the api/v1/access-group/my endpoint

### PR DESCRIPTION
**Pull request by ivaylomitrev**

### CODE REVIEW

**Review by venelinkotsev — APPROVED** [review]

**Review by phillipj — APPROVED** [review]

**Comment by phillipj** [comment]
@ivaylomitrev this is probably good to be merged now?

**Comment by ivaylomitrev** [comment]
@phillipj , the only reason I haven't merged it yet is because the version of Documaster it will be introduced in has not yet been released.

**Comment by phillipj** [comment]
Ohh alrighty, makes sense 👍

File: v1/endpoints.md [hunk 1]
```diff
@@ -1029,6 +1030,61 @@ Response codes:
...
+### Request
+
+```
+GET /access-group/my?page=1&pageSize=10 HTTP/1.1
+Accept: application/json
+Authorization: Bearer {token}
+```
+
+- `page` (optional)
+  - specifies the page to fetch; 1-based
```
Comment (vulkodava) [inline v1/endpoints.md:1069]: Should 1-based be on a separate line? As defaults to 1 is?

Comment (ivaylomitrev) [reply v1/endpoints.md:1069]: I don't have an opinion on that to be honest. I can change it if you'd like, but to me it's equivalent in terms of readability/understanding.

Comment (vulkodava) [reply v1/endpoints.md:1069]: Doesn't look to be necessary :) to me too.

**Pull request merged**